### PR TITLE
Add kaiju2table to taxprofiler

### LINKED ISSUE #3: Profiler request: kaiju
### Description of feature

Kaiju is a program for sensitive taxonomic classification of high-throughput sequencing reads from metagenomic whole genome sequencing or metatranscriptomics experiments

https://kaiju.binf.ku.dk/

### PR DESCRIPTION
**Pull request by sofstam**
This PR adds the [kaiju/kaiju2table](https://github.com/nf-core/modules/tree/master/modules/kaiju/kaiju2table) module to taxprofiler and the output is parsed by MultiQC. It closes #3 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -384,6 +384,10 @@
         },
         "malt_generatemegansummary": {
             "type": "boolean"
+        },
+        "kaiju_taxon_name": {
+            "type": "string",
+            "default": "species"
```
Comment (jfy133) [inline nextflow_schema.json:390]: Please give the range of taxa using the enumerated list option (accessable in `schema build` by pressing I think the gear/cog icon to the right of the given param entry)

```
   -r STRING     Taxonomic rank, must be one of: phylum, class, order, family, genus, species

```

**Review by jfy133 — APPROVED** [review]
just one thing to update in the `json schema` and then you are free to merge :)

File: nextflow_schema.json (hunk 1 above)
Comment (sofstam) [reply nextflow_schema.json:390]:
```
"kaiju_taxon_name": {
            "type": "string",
            "default": "species",
            "enum": [
                "phylum",
                "class",
                "order",
                "family",
                "genus",
                "species"
            ]
```

Shall I remove the default value then, right?

Comment (jfy133) [reply nextflow_schema.json:390]: No keep the default, I think that's fine

**Review by jfy133 — APPROVED** [review]

**Comment by sofstam** [comment]
I will merge once the prettier tests pass.

**Pull request merged**